app/controllers/admin/courses_controller.rb

```diff
@@ -118,7 +118,7 @@ def course_params
                                      :contributor,  :pub_status,      :language_id,     :level,             :topics,
                                      :notes,        :delete_document, :other_topic,     :other_topic_text,  :course_order,
                                      :pub_date,     :format,          :subsite_course,  :display_on_dl,     :subdomain,
-            attachments_attributes: [:course_id, :document, :title, :doc_type, :_destroy])
+            attachments_attributes: [:course_id, :document, :title, :doc_type, :_destroy, :file_description])
```

Comment:
I don't think it's absolutely critical, but I think it's most common to put ```:_destroy``` at the end of the list of params. I seem to remember hearing something about ordering of params sometimes causing issues, and I don't know if it would be related to this, but just a thought. If you've tested that destroying does work as intended though, it's probably fine.